### PR TITLE
Minor changes, see description

### PR DESCRIPTION
**Pull request by dan-michell**
Changed profile name, updated search tab selector, added tooltip to microphone

### CODE REVIEW

File: src/GlobalComponents/Header/Header.js [hunk 1]
```diff
@@ -150,7 +150,7 @@ export default function Header() {
       return (
         <Box sx={{ flexGrow: 1, display: { xs: "none", md: "flex" } }}>
           <Button
-            key="profile"
```
Comment (Ibahmed1) [inline src/GlobalComponents/Header/Header.js:153]: genius change

File: src/Home/Home.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ import MealResultCard from "./components/MealResultCard";
 import SpeechDetection from "./components/SpeechDetection";
 
 export default function Home(props) {
-  const [tab, setTab] = useState("Product name");
+  const [tab, setTab] = useState("Text");
```
Comment (Ibahmed1) [inline src/Home/Home.js:15]: outstanding work daniel

Comment (dan-michell) [reply src/Home/Home.js:15]: Cheers boss

File: src/Home/Home.js [hunk 3]
```diff
@@ -134,7 +135,12 @@ export default function Home(props) {
           ></TextField>
           <div className="search-icons">
             <Tooltip title="Search">
-              <IconButton data-testid="search-button" aria-label="Search..." color="primary" onClick={() => handleSearch()}>
+              <IconButton
+                data-testid="search-button"
+                aria-label="Search..."
+                color="primary"
+                onClick={() => handleSearch()}
+              >
```
Comment (Ibahmed1) [inline src/Home/Home.js:143]: zuckerberg would be in awe

Comment (dan-michell) [reply src/Home/Home.js:143]: Stop that

File: src/Home/components/TabSelector.js [hunk 4]
```diff
@@ -36,15 +38,15 @@ export default function TabSelector(props) {
 
   const handleChange = (event, newValue) => {
     setValue(newValue);
-    const tab = newValue === 0 ? "Product name" : "Barcode";
+    const tab = newValue === 0 ? "Text" : "Barcode";
```
Comment (Ibahmed1) [inline src/Home/components/TabSelector.js:41]: I think there is a better way to do this

**Pull request merged**